docs/api/beta.md

```diff
@@ -30,47 +30,16 @@ Our APIs are built with [OpenAPI](https://www.openapis.org/). You can generate c
 
```

Comment:
Why is this article in beta? It's been that way for as long as I've been here. Just curious.

Reply:
Great catch! Most of them still actually are in beta, but I'll update the ones that GA'd and then resubmit this PR.

Reply:
This one should stay in beta:
https://help.sumologic.com/docs/manage/users-roles/roles/rbac-for-indexes/

GA is projected to be April 30.